src/_cffi_src/openssl/nid.py

```diff
@@ -25,12 +28,33 @@
 
 static const int NID_subject_alt_name;
 static const int NID_crl_reason;
+static const int NID_INN;
+static const int NID_OGRN;
+static const int NID_SNILS;
 """
 
 FUNCTIONS = """
 """
 
 CUSTOMIZATIONS = """
+#ifndef NID_INN
```

Comment:
Do we need these definitions at all?

Reply:
Yes we need this definitions. This OIDS (INN/SNILS/ORGN) is required to obtain status of qualified certificate in Russian Federation. Binding this OIDs is usefull

Reply:
But this is just binding things from the C layer that we do not actually use since we're separately defining the OIDs, right?

Reply:
ok

Reply:
@reaperhulk declarations removed